appium/options/android/common/adb/adb_options.py

```diff
@@ -0,0 +1,367 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from appium.options.base_options_descriptor import OptionsDescriptor
+from appium.options.transformers import DurationTransformer
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+
+class AdbExecTimeoutOption(SupportsCapabilities):
+    ADB_EXEC_TIMEOUT = 'adbExecTimeout'
+    adb_exec_timeout = OptionsDescriptor(
```

Comment:
generic type definitions are still missing for all OptionsDescriptor definitions. Did you run mypy on the source?

Reply:
Fixed. Added type hints to all OptionsDescriptor. attaching screen shot of mypy results.

 
<img width="802" alt="mypy" src="https://github.com/appium/python-client/assets/26169602/5099abbf-cb46-406d-867b-24fc922433ff">